### PR TITLE
Sorting of references

### PR DESCRIPTION
**Pull request by Daskyleia**
Resolving issue #16

### CODE REVIEW

File: source/references.adoc [hunk 1]
```diff
@@ -1,19 +1,26 @@
 [appendix]
 = References
 
-Below are references to various topics of interest around RPMs, RPM Packaging,
-and RPM Building. Some of these will be advanced and extend far beyond the
-introductory material included in this guide.
+Below are references to various topics of interest around RPMs, RPM Packaging, and RPM Building. Some of these will be advanced and extend far beyond the introductory material included in this guide.
```
Comment (pmkovar) [inline source/references.adoc]: Below are references to various topics of interest around RPMs, RPM packaging, and RPM building. Some of these are advanced and extend far beyond the introductory material included in this guide.

File: source/references.adoc [hunk 2]
```diff
@@ -1,19 +1,26 @@
...
+
+ifdef::rhel[]
+
+*   https://developers.redhat.com/products/softwarecollections/overview/[Red Hat Software Collections] - The Red Hat Software Collections provides continuously updated development tools in latest stable versions.
```
Comment (pmkovar) [inline source/references.adoc]: link:https://developers.redhat.com/products/softwarecollections/overview/[Red Hat Software Collections] - The Red Hat Software Collections offering provides continuously updated development tools in latest stable versions.

File: source/references.adoc [hunk 3]
```diff
@@ -1,19 +1,26 @@
...
+*   https://rpm-packaging-guide.github.io/#mock[Mock] - Mock provides package building solution for various architectures and different Fedora or RHEL versions than has the build host.
+ 
+endif::rhel[]
 
-*   http://rpm.org/wiki/Docs[The Official RPM Documentation]
-*   https://www.gurulabs.com/media/files/courseware-samples/GURULABS-RPM-GUIDE-v1.0.PDF[Gurulabs CREATING RPMS (Student Version)]
 ifdef::community[]
-*   https://fedoraproject.org/wiki/How_to_create_an_RPM_package[Fedora How To Create An RPM Package Guide]
+
+*   https://www.softwarecollections.org/en/[Software Collections] - SoftwareCollections.org is the home for projects creating Software Collections (SCLs) for Red Hat Enterprise Linux, Fedora, CentOS, and Scientific Linux.
```
Comment (pmkovar) [inline source/references.adoc]: link:https://www.softwarecollections.org/en/[Software Collections] - SoftwareCollections.org is an open-source project for building and distributing community-supported Software Collections (SCLs) for Red Hat Enterprise Linux, Fedora, CentOS, and Scientific Linux.

File: source/references.adoc [hunk 4]
```diff
@@ -1,19 +1,26 @@
...
-*   https://en.opensuse.org/openSUSE:Packaging_guidelines[OpenSUSE Packaging Guidelines]
-*   IBM RPM Packaging Guide: http://www.ibm.com/developerworks/library/l-rpm1/[Part 1], http://www.ibm.com/developerworks/library/l-rpm2/[Part 2], http://www.ibm.com/developerworks/library/l-rpm3/[Part 3]
-*   http://ftp.rpm.org/max-rpm/[Maximum RPM] (a good resource for advanced topics, even though some material is obsolete)
-ifdef::rhel[] 
-*   https://rpm-packaging-guide.github.io/#mock[Mock] 
-endif::rhel[]
+
+*   http://rpm.org/documentation[RPM Documentation] - The official RPM documentation.
+
+*   https://fedoraproject.org/wiki/Packaging:Guidelines?rd=Packaging/Guidelines[Fedora Packaging Guidelines] - Packaging guidelines for Fedora useful for all RPM based distributions.
```
Comment (pmkovar) [inline source/references.adoc]: link:https://fedoraproject.org/wiki/Packaging:Guidelines?rd=Packaging/Guidelines[Fedora Packaging Guidelines] - The official packaging guidelines for Fedora, useful for all RPM-based distributions.

File: source/references.adoc [hunk 5]
```diff
@@ -1,19 +1,26 @@
...
-Below are references to various topics of interest around RPMs, RPM Packaging,
-and RPM Building. Some of these will be advanced and extend far beyond the
-introductory material included in this guide.
+Below are references to various topics of interest around RPMs, RPM packaging, and RPM building. Some of these are advanced and extend far beyond the introductory material included in this guide.
+
+ifdef::rhel[]
+
+link:https://developers.redhat.com/products/softwarecollections/overview/[Red Hat Software Collections] - The Red Hat Software Collections is continuously offering updated development tools in latest stable versions.
+
+link:https://rpm-packaging-guide.github.io/#mock[Mock] - Mock provides package building solution for various architectures and different Fedora or RHEL versions than has the build host.
```
Comment (pmkovar) [inline source/references.adoc]: Mock provides a community-supported package building solution...

File: source/references.adoc [hunk 6]
```diff
@@ -1,19 +1,26 @@
...
+ 
+endif::rhel[]
 
-*   http://rpm.org/wiki/Docs[The Official RPM Documentation]
-*   https://www.gurulabs.com/media/files/courseware-samples/GURULABS-RPM-GUIDE-v1.0.PDF[Gurulabs CREATING RPMS (Student Version)]
 ifdef::community[]
-*   https://fedoraproject.org/wiki/How_to_create_an_RPM_package[Fedora How To Create An RPM Package Guide]
+
+link:https://www.softwarecollections.org/en/[Software Collections] - SoftwareCollections.org is an open-source project for building and distributing community-supported Software Collections (SCLs) for Red Hat Enterprise Linux, Fedora, CentOS, and Scientific Linux.
```
Comment (pmkovar) [inline source/references.adoc]: Let's also add a Red Hat supported counterpart (RHSCL), under ifdef::rhel[]:

https://access.redhat.com/documentation/en-us/red_hat_software_collections/3/html/packaging_guide/

Best to use parts of the Packaging Guide's abstract as the link description.

Comment (Daskyleia) [reply source/references.adoc]: Updated according to peer review.

File: source/references.adoc [hunk 7]
```diff
@@ -1,19 +1,28 @@
...
-Below are references to various topics of interest around RPMs, RPM Packaging,
-and RPM Building. Some of these will be advanced and extend far beyond the
-introductory material included in this guide.
+Below are references to various topics of interest around RPMs, RPM packaging, and RPM building. Some of these are advanced and extend far beyond the introductory material included in this guide.
+
+ifdef::rhel[]
+
+link:https://developers.redhat.com/products/softwarecollections/overview/[Red Hat Software Collections Overview] - The Red Hat Software Collections offering provides continuously updated development tools in latest stable versions.
+
+link:https://access.redhat.com/documentation/en-us/red_hat_software_collections/3/html/packaging_guide/ [Red Hat Software Collections] - The Packaging Guide provides an explanation of Software Collections and details how to build and package them.  Developers and system administrators with basic understanding of software packaging with RPM can use this Guide to get started with Software Collections.
```
Comment (pmkovar) [inline source/references.adoc]: The extra space between `packaging_guide/` and `[Red` breaks formatting, I think.

File: source/references.adoc [hunk 8]
```diff
@@ -1,19 +1,28 @@
...
-*   http://rpm.org/wiki/Docs[The Official RPM Documentation]
-*   https://www.gurulabs.com/media/files/courseware-samples/GURULABS-RPM-GUIDE-v1.0.PDF[Gurulabs CREATING RPMS (Student Version)]
 ifdef::community[]
-*   https://fedoraproject.org/wiki/How_to_create_an_RPM_package[Fedora How To Create An RPM Package Guide]
+
+link:https://www.softwarecollections.org/en/[Software Collections] - SoftwareCollections.org is an open-source project for building and distributing community-supported Software Collections (SCLs) for Red Hat Enterprise Linux, Fedora, CentOS, and Scientific Linux.
+
+link:https://docs.fedoraproject.org/quick-docs/en-US/creating-rpm-packages.html[Creating RPM package] - Step-by-step guide for learning basics of RPM packaging.
+
+link:http://www.ibm.com/developerworks/library/l-rpm1/[Part 1], http://www.ibm.com/developerworks/library/l-rpm2/[Part 2], http://www.ibm.com/developerworks/library/l-rpm3/[Part 3] - IBM RPM packaging guide.
```
Comment (pmkovar) [inline source/references.adoc]: s/Part 1/Packaging software with RPM, Part 1/

Same for Part 2.

**Pull request merged**